docs/product/alerts/create-alerts/metric-alert-config.mdx

```diff
@@ -132,11 +132,11 @@ Change alerts, or alerts that use a percent change threshold, are useful when yo
 <Include name="feature-available-for-plan-trial-business.mdx" />
 <Include name="feature-available-for-user-group-early-adopter.mdx" />
 
-Anomaly alerts can be used when you don't know what threshold to set, but you know you want to be alerted when something is far outside the bounds of normalcy. Sentry will look at the historical data for the given metric and determine if the current data is anomalous. You have three options for the responsiveness of the alert: low, medium, and high. Low responsiveness means the alert will fire less frequently with each instance being higher signal. High responsiveness will capture more instances of the alert, which creates more coverage with a greater likelihood for false positives. Our recommendation is to start with our default (medium), and iterate based on the results you see.
-Another setting is to set the direction of the alert: above and/or below the expected bounds. This setting can help increase the signal of your alert rule.
+Anomaly alerts automatically detect trends outside expected values. They can be especially helpful for spiky or seasonal data that are too noisy for static or percentage based thresholds. Sentry will look at historical data for the given metric and determine if the current data is anomalous. Apdex is currently not supported for anomaly alerts.
```

Comment:
> Apdex is currently not supported for anomaly alerts.

I'd prefer to keep this vague, another team recently added some other new types of metrics and I think there might be a couple others that aren't supported. If it's not supported, the dynamic threshold option won't be shown in the UI.

Reply:
okay, I'll try some different language here